.release-it.json

```diff
@@ -1,9 +1,9 @@
 {
     "hooks": {
-        "after:bump": "./.github/helpers/update-saucectl-template.sh"
+        "after:bump": "bash ./scripts/update-playwright-browsers.sh"
```

Comment:
Might as well delete this file https://github.com/saucelabs/sauce-playwright-runner/blob/master/.github/helpers/update-saucectl-template.sh altogether, since we don't use templates any longer. Along with the steps in the pipeline that attach the template :)

Reply:
Removed all the template stuff, fyi.